src/libraries/Common/src/Interop/Windows/Advapi32/Interop.ConvertStringSidToSid.cs

```diff
@@ -8,13 +8,8 @@ internal static partial class Interop
 {
     internal static partial class Advapi32
     {
-#if DLLIMPORTGENERATOR_ENABLED
-        [GeneratedDllImport(Interop.Libraries.Advapi32, EntryPoint = "ConvertStringSidToSidW", CharSet = CharSet.Unicode, SetLastError = true)]
+        [GeneratedDllImport(Interop.Libraries.Advapi32, EntryPoint = "ConvertStringSidToSidW", CharSet = CharSet.Unicode, ExactSpelling = true, SetLastError = true)]
```

Comment:
Was the addition of ExactSpelling here just an opportunistic addition?  Should we be using it in other places we're not currently

Reply:
Yeah, this was just opportunistic - I just noticed that already had the `W` suffix while doing this.